controllers/argo.go

```diff
@@ -155,8 +167,7 @@ func commonSyncPolicy(p api.Pattern) *argoapi.SyncPolicy {
 			// Options allow you to specify whole app sync-SyncOptions
 			SyncOptions: []string{"Prune=true"},
 		}
-
-	} else if !p.Spec.GitOpsConfig.ManualSync {
+	} else if !commonSyncPolicyFromConfigMap() {
```

Comment:
Did you discuss with @mbaldessari if this needs to be a per-pattern value?

Reply:
Yes. And @mbaldessari mentioned that we would circle back to this.  For now we decided to leave it as is.

Reply:
This has been updated and we now include this structure in the CRD with one element.